.github/workflows/build-lambda-zip.yml

```diff
@@ -0,0 +1,27 @@
+name: go get build-lambda-zip 
```

Comment:
Looks like you can specify a [description](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/metadata-syntax-for-github-actions#description). Imo it's a bit unclear *why* this action exists and perhaps a short explanation in the description would be beneficial?